boards/arm/nucleo_f030r8/nucleo_f030r8.dts

```diff
@@ -41,6 +41,11 @@
 	};
 };
 
+/* Due to limited available memory, don't enable gpiod and gpiof */
+/* (Test cases fail due to 'SRAM' region overflow) */
+&gpiod {status = "disabled";};
```

Comment:
Any reason for this style vs:
```
&gpiod {
    status = "disabled";
};
```

Reply:
I found this easier to read, specially when you start having 3 of those.